src/integration-test/java/apptest/ReadDetailsIT.java

```diff
@@ -104,7 +86,7 @@ void test5_readDetailsByCustomerEngagementOrgIdNotFound() throws Exception {
 	}
 
 	@Test
-	void test6_readDetailsByCustomerEngagementOrgIdWithFromDate() throws Exception {
+	void test5_readDetailsByCustomerEngagementOrgIdWithFromDate() throws Exception {
 		final var result = setupCall()
```

Comment:
Why are you returning a body in this method?

You have: 
`.withExpectedResponse("response.json")`

Doesn't this cover all response verifications?

Reply:
Good question, the tests were already written and I didn't reflect on the additional asserts, I'll remove them